compiler/rustc_resolve/Cargo.toml

```diff
@@ -21,6 +21,7 @@ rustc_expand = { path = "../rustc_expand" }
 rustc_feature = { path = "../rustc_feature" }
 rustc_hir = { path = "../rustc_hir" }
 rustc_index = { path = "../rustc_index" }
+rustc_lint_defs = { path = "../rustc_lint_defs" }
```

Comment:
`rustc_resolve` typically uses lints through `rustc_session::lint`, so adding this shouldn't be necessary.